src/main/java/Duke.java

```diff
@@ -1,10 +1,188 @@
+import java.time.LocalTime;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.io.IOException;
+
 public class Duke {
+    private static void format(String output) {
+        String border = "   ____________________________________________________________";
+        System.out.println(border + "\n      " + output + "\n" +  border);
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        final String SPACE = "\n      ";
+        final String TASKSPACE = "\n        ";
+        String logo =
+                " _____           _ _   \n" +
+                "|  ___|         (_) |   \n" +
+                "| |__ _ __   ___ _| | __\n" +
+                "|  __| '_ \\ / __| | |/ /\n" +
+                "| |__| | | | (__| |   < \n" +
+                "\\____/_| |_|\\___|_|_|\\_\\\n";
+        System.out.println("GOOD MORNING GENNERMEN from\n" + logo);
+
+        String border = "   ____________________________________________________________";
+        String welcome =
+                "GOOD MORNING CHAO RECRUIT! YOU MAY CALL ME ENCIK ENCIK" + SPACE +
+                "WHAT YOU WANT?";
+        format(welcome);
+
+        //Checks if prev. tasks exist
+        String logPath = "data/log.txt";
+        DukeFile log = new DukeFile(logPath);
+
+        //Create ArrayList to store prev tasks
+        ArrayList<Task> tasks = log.readTasks();
+
+        Scanner input = new Scanner(System.in);
+        String toEcho = input.nextLine();
+        while (!toEcho.equals("bye")) {
+            try {
+                //Add name of task to str for easy printing
+                StringBuilder str = new StringBuilder();
+                str.append("NEED ME TO REMIND YOU AH?");
+                if (toEcho.equals("list")) {
+                    int size = tasks.size();
+                    if (size == 0) {
+                        format("NOTHING TO DO AH?" + SPACE +
+                                "YOU BETTER FIND SOMETHING TO DO BEFORE I CONFINE YOU CHAO RECRUIT!");
+                    } else {
+                        for (int i = 0; i < size; i++) {
+                            str.append(SPACE + (i + 1) + "." + tasks.get(i));
+                        }
+                        format(str.toString());
+                    }
+                } else {
+                    //Split cmd into 2 parts, the type of task, and remaining text
+                    String[] temp = toEcho.split(" ", 2);
+                    String cmd = temp[0];
+                    //placeholder in case temp[1] is empty and cannot be initialised
+                    String rem = "";
+                    boolean incomplete = false;
```

Comment:
I believe the boolean name needs to be changed according to the coding standard.

![image](https://user-images.githubusercontent.com/63341400/151937900-9bdaca84-bbb3-49ee-ab97-6e0c8c5e4112.png)


```suggestion
                    boolean isIncomplete = false;
```